code/js/modules/BaseController.js

```diff
@@ -43,7 +43,13 @@
       song: null,
       artist: null,
       isPlaying: null,
-      siteName: null
+      siteName: null,
```

Comment:
I gave it some thought, and I think defining the properties here is just too verbose (even for my standards).

Please change this block so line 42 is just `this.oldState = {};`